customTests/azure_nccl_allreduce_ib_loopback.nhc

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Check for IB issues by running NCCL allreduce disabling NCCL shared memory.
+# Expected performance is > 19 GB/s
+
+TOPOFILE=$3
+if [[ -z "$TOPOFILE" ]];then
+   TOPOFILE=/opt/microsoft/ndv4-topo.xml
+fi
+
+MPI_ARGS="-np 8 --map-by ppr:8:node -bind-to numa -mca coll_hcoll_enable 0 --allow-run-as-root"
+ENVIRON_VARS="-x LD_LIBRARY_PATH=/usr/local/nccl-rdma-sharp-plugins/lib:$LD_LIBRARY_PATH -x NCCL_IB_PCI_RELAXED_ORDERING=1 -x UCX_IB_PCI_RELAXED_ORDERING=on -x UCX_TLS=tcp -x UCX_NET_DEVICES=eth0 -x CUDA_DEVICE_ORDER=PCI_BUS_ID -x NCCL_SOCKET_IFNAME=eth0 -x NCCL_NET_GDR_LEVEL=5 -x NCCL_TOPO_FILE=$TOPOFILE -x NCCL_SHM_DISABLE=1 -x NCCL_P2P_DISABLE=1"
+NCCL_ARGS="-b 500M -f 2 -g 1 -e 1G -c 1"
```

Comment:
Why are we using this size rather than 4G?

Reply:
addressed offline